src/mod/dns.mod/coredns.c

```diff
@@ -823,11 +815,12 @@ void parserespacket(uint8_t *response, int len)
   int ready = 0;
 #endif
   int r, rcount;
-  res_record *rr;
   packetheader *hdr;
   struct resolve *rp;
   uint8_t rc, *c = response;
-  uint16_t qdatatype, qclass;
+  uint16_t qdatatype, qclass, datatype, class, datalength;
```

Comment:
"class" is a variable name that won't make our code compileable or testable with c++ (cpputest for example), could you change it to aclass? I don't generally pay attention to that but "class" is just a very obvious one

Reply:
This would be a bug in cpputest or testing method. Eggdrop is C, and any test program must respect this. What if the next test program is python or javascript? There is not even evidence that cpputest is used and would fail. so i would like to reject your change request this time.